crates/wash-cli/src/bin/wash.rs

```diff
@@ -511,12 +533,23 @@ async fn main() {
         std::process::exit(0);
     };
 
+    if cli.version {
+        let version = version(cli.output);
+        println!("{}", version);
+        std::process::exit(0);
+    }
+
+    let cli_command = cli.command.unwrap_or_else(|| {
+        eprintln!("{}", help);
```

Comment:
```suggestion
        eprintln!("{}", command.render_help());
```

I think we can defer rendering the help until someone unless we fail to unwrap the command?